receiver/vmmetricsreceiver/metrics_receiver.go

```diff
@@ -107,3 +108,21 @@ func (vmr *Receiver) StopMetricsReception(ctx context.Context) error {
 	})
 	return err
 }
+
+// TODO(songya): investigate why viper.Unmarshal didn't work, remove this method and use viper.Unmarshal instead.
```

Comment:
At first this looks strange, perhaps is it because the suffix of two entries are matching?

Reply:
Actually even if I removed both *_mount_point configs Viper still failed to parse the config. I'll investigate and fix it in a separate PR.